Makefile

```diff
@@ -351,7 +351,8 @@ build-all-new-commits:
 vendor:
 	vndr -whitelist "github.com/varlink/go"  \
 	     -whitelist "github.com/onsi/ginkgo" \
-	     -whitelist "github.com/onsi/gomega"
+	     -whitelist "github.com/onsi/gomega" \
+		 -whitelist "gopkg.in/fsnotify.v1"
 
 .PHONY: \
 	.gopathok \
```

Comment:
might have a merge dance here with @valentin makefile vendor changes.

Reply:
this was done with his instruction

Reply:
Your indentation is off here